docs/apis/cli.md

```diff
@@ -185,6 +189,18 @@ Action "list" lists running and scheduled programs.
      -s,--scheduled                Show only scheduled programs and their JobIDs
 
 
+Action "stop" stops a running program (streaming jobs only).
+
+  Syntax: stop [OPTIONS] <Job ID>
+  "stop" action options:
+     -m,--jobmanager <host:port>   Address of the JobManager (master) to which
+                                   to connect. Specify 'yarn-cluster' as the
+                                   JobManager to deploy a YARN cluster for the
+                                   job. Use this flag to connect to a different
```

Comment:
I understand. Makes sense. I just c&p from other commands (and was wounding about it anyway ;) -- maybe I should have asked right away). The option `-m` in also specified for `cancel`, `list`. I guess it requires a complete clean up. Should I just include this in this PR or should be open a JIRA (and I just leave this inconsistency as is here)?